db/migrate/20181001163831_create_active_admin_comments.rb

```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
```

Comment:
Wait, what is this?

Reply:
I something that the gem created automatically when i run "rails g active_admin:install"

Reply:
*ran